packages/ui-color-picker/src/ColorPreset/index.tsx

```diff
@@ -272,6 +272,7 @@ class ColorPreset extends Component<ColorPresetProps, ColorPresetState> {
               />
             </div>
           )}
+          <ScreenReaderContent>{`Color: ${color}`}</ScreenReaderContent>
```

Comment:
This makes the screen reader read two labels (from the tooltip plus this one). Since this another case where the screen reader reads it well, because we solved it with a workaround, and the a11y check is not smart enough to see it, I would just ignore the `button-name` error in the a11y tests.